### PR TITLE
feature/JP-6-login

### PR DESCRIPTION
**Pull request by Ixpiria**

### CODE REVIEW

File: frontend/components/svg/cloudReversed.vue
Comment (EinSev) [inline frontend/components/svg/cloudReversed.vue:1]: Please rename to "cloud-reversed.vue"

File: frontend/pages/login.vue [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+});
+
+interface User {
+  email: string;
+  password: string;
+}
+/**
+ * use the sanctum client to login a user
+ * shows toast messages if success/error
+ * @param {Object} userData
```
Comment (EinSev) [inline frontend/pages/login.vue]: @param Object? I think its User

File: frontend/pages/login.vue [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    email: yup
+      .string()
+      .email(t.value("form.input.email.error"))
+      .required(t.value("form.input.required")),
+    password: yup.string().required(t.value("form.input.required")),
+  }),
+});
+
+const onSubmit = handleSubmit((values) => {
+  loginUser(values);
```
Comment (EinSev) [inline frontend/pages/login.vue]: Please fix TypeScript error:

```
Argument of type 'GenericObject' is not assignable to parameter of type 'User'.
  Type 'GenericObject' is missing the following properties from type 'User': email, passwordts-plugin(2345)
(parameter) values: GenericObject
```

File: frontend/pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        class="xl:w-[50%] md:w-[80%] w-0 object-none overflow-hidden mt-[20vh] z-50 dark:fill-clouds-bg -ml-24"
+      />
+      <SvgCloud
+        class="absolute xl:w-[20%] md:w-[25%] md:left-[6vh] left-[23vh] w-0 object-none overflow-hidden top-[49vh] z-30"
+      />
+      <div class="overflow-hidden">
+        <SvgCloudReversed
+          class="xl:w-[45%] md:w-[70%] w-0 object-none overflow-hidden mt-[55vh] z-50 dark:fill-clouds-bg"
+        />
+        <SvgBalloon
```
Comment (EinSev) [inline frontend/pages/login.vue:93]: Something went wrong exporting these svgs, they have the wrong color.

File: frontend/pages/login.vue [hunk 4]
```diff
@@ -0,0 +1,151 @@
+<script setup lang="ts">
+import { useForm } from "vee-validate";
+import { useTranslate, T, useTolgee } from "@tolgee/vue";
```
Comment (EinSev) [inline frontend/pages/login.vue]: Remove unused import

**Review by EinSev — CHANGES_REQUESTED** [review]

**Review by EinSev — DISMISSED** [review]

**Review by stefll — APPROVED** [review]
Design Review: Okay
Habt ihr super gemacht

**Review by EinSev — APPROVED** [review]

**Pull request merged**